### DISCUSSION TITLE
Client Id and Client secret to access NHCX sandbox APIs

### DISCUSSION THREAD

**Discussion by VarunnSingla** [NHA-ABDM/nhcx#18] [Q&A] [answered]
How do I obtain a Client ID and Secret to access the APIs within the NHCX sandbox?

**Comment by vishi24** [comment, marked as answer]
You can use the same credentials that were used in M1.
Once your application is on M1 production, register on NHCX sandbox.
The Sandbox team will attach NHCX roles to your M1 sandbox credentials, which can be used on the NHCX sandbox to access the APIs.
